libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -10,7 +10,8 @@ VotePacketHandler::VotePacketHandler(std::shared_ptr<PeersState> peers_state,
                                      std::shared_ptr<VoteManager> vote_mgr, const addr_t &node_addr)
     : ExtVotesPacketHandler(std::move(peers_state), std::move(packets_stats), node_addr, "PBFT_VOTE_PH"),
       pbft_mgr_(std::move(pbft_mgr)),
-      vote_mgr_(std::move(vote_mgr)) {}
+      vote_mgr_(std::move(vote_mgr)),
+      known_votes_(1000000, 1000) {}
```

Comment:
kind of subjective but `seen_votes_`  seems like a better expression considering what this cache actually contains....